libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/cellscanner/CellScanner.java

```diff
@@ -128,8 +129,6 @@ private synchronized void addToCells(String cell) {
     }
 
     public synchronized void stop() {
-        mReportWasFlushed.set(false);
-        clearCells();
```

Comment:
isn't clearCells() still needed? If the scanning starts again, the old cells will still be in the list (I think)